libs/ui/ng-package.json

```diff
@@ -2,7 +2,8 @@
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/libs/ui",
   "lib": {
-    "entryFile": "src/index.ts"
+    "entryFile": "src/index.ts",
+    "styleIncludePaths": ["../styles"]
```

Comment:
This lets us import cvi-styles to components of cvi-ng. But it might not be the right thing to do given that a consumer app written in Angular and using cvi-ng might not have cvi-styles installed (although without the styles Angular components are not styled and should be useless anyway).